modules/oc-mirror-enclave-support-about.adoc

```diff
@@ -19,4 +19,35 @@ Enclave support has the following benefits:
 
 * You can mirror content directly from the centralized internal registry to enclaves without restarting the mirroring process from the internet for each enclave.
 
-* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
\ No newline at end of file
+* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
+
+== How to mirror to an enclave
```

Comment:
Also I think this title might be a little misleading because this section is followed by a page for "mirroring to an enclave", which sounds like a similar concept to "how to mirror an enclave". I don't know what a good alternative would be, but I would suggest you think of one to replace this heading with. Here are a few ideas, though I don't know how accurate they'd be:

```suggestion
== Enclave mirroring workflow
```

```suggestion
== About enclave mirroring
```